src/test/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImplTest.java

```diff
@@ -1122,6 +1129,22 @@ public void setClient(Client client) {
             this.mockClient = client;
         }
         
+        @Override
+        protected Invocation buildClientRequest(String url, Collection<WebServiceCredential> creds) throws URISyntaxException {
+            if (StringUtils.equals(url, 

```

Comment:
Jay and I chatted about changing the override to use a boolean instead of checking on the specific URL